playbooks/ansible/roles/sit.cephfs/tasks/common/centos.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Identify latest ceph:{{ config.data.branch }} build
+  uri:
+    url: "https://shaman.ceph.com/api/search?\
+            project=ceph&\
+            ref={{ config.data.branch }}&\
+            flavor=default&\
+            status=ready&\
+            distros=centos/{{ config.os[site.os].version }}/x86_64&\
+            sha1=latest"
+    return_content: true
+  register: ceph_shaman
+
+- name: Create ceph repo
+  yum_repository:
+    name: "ceph-{{ config.data.branch }}-{{ ceph_shaman.json[0].sha1 }}"
+    baseurl: "{{ ceph_shaman.json[0].url }}/x86_64"
+    description: "Ceph {{ config.data.branch }} ({{ ceph_shaman.json[0].sha1 }})"
+    enabled: true
+    gpgcheck: false
+    state: present
```

Comment:
Why is this needed for client machines ?

Reply:
_smbtorture_ is the requirement here on client machines. Thus in case of CephFS we need to have necessary development packages installed while configuring and installing samba for `smbtorture`.

Reply:
But in what way smbtorture depends on CephFS ? smbtorture is just a Samba client.

Is it because we build the client side Samba with same configuration as the servers ?

Reply:
Yes, same role is used for both client and server.

Reply:
Ok